source/docs/installation/non-frc-checklist.rst

```diff
@@ -0,0 +1,91 @@
+:orphan:
```

Comment:
Can this be visible under the Installing Phoenix 6 tab?
Just like how Tuner X has the "Connecting Tuner" section
![image](https://github.com/CrossTheRoadElec/PhoenixPro-Documentation/assets/28712271/f31e6702-5b88-4166-93e3-34c82ce7ed6d)